scripts/updateSchema/schema.graphql

```diff
@@ -0,0 +1,42 @@
+enum AccessLevel {
+  ADMIN
+  MANAGER
+  WORKER
+}
+
+type User {
+  name:String
+  email:String! @id
+  picture:String
+}
+
+type UserRole {
+  user:User
+  level:AccessLevel
```

Comment:
Here is more or less the same with Workspace (I encourage you to call only Workspace) 



type UserRole {
    user: User
    level: AccessLevel
    workspace: Workspace
}

type Workspace {
    name: String
    users: [UserRole] @hasInverse(field:workspace)
}